verify-scripts.js

```
@@ -0,0 +1,41 @@
+}
+
+// Verify that start:mock includes the test mode flag
+if (packageJson.scripts['start:mock'] && packageJson.scripts['start:mock'].includes('--test-mode')) {
+  console.log('✅ npm run start:mock correctly runs in MOCK mode (with test flag)');
+} else {
+  console.log('❌ npm run start:mock does not exist or doesn\'t use test mode flag');
+}
+
+// Verify that dev script references start:mock
+if (packageJson.scripts.dev && packageJson.scripts.dev.includes('start:mock')) {
+  console.log('✅ npm run dev correctly references start:mock');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)